hellosign_sdk/tests/unit_tests/test_hsformat.py

```diff
@@ -42,7 +42,7 @@ def test_format_file_params(self):
 
         keys = result.keys()
 
-        attribute = hasattr(result[keys[0]], 'read') #check if file (can be read)
+        attribute = hasattr(result[list(keys)[0]], 'read') #check if file (can be read)
```

Comment:
Good catch, `.keys()` returns a `dict_keys` object in Python 3. 

Wondering if we want to call `list()` in line 43 (eg: `key = list(result.keys())`) that way the `key` variable is a list and maintains the same behavior as in Python 2.7

Reply:
Yeah, on second thought I think it will be easier to read if we do it in line `43` when we declare the `keys` variable.